### PR TITLE
fix/cache-rollen

### PR DESCRIPTION
**Pull request by damm89**

### CODE REVIEW

File: backend/src/zac/core/cache.py [hunk 1]
```diff
@@ -71,11 +71,7 @@ def invalidate_document_cache(document: Document):
...
 
-def invalidate_rollen_cache(zaak: Zaak, rollen: Optional[List[str]] = None):
+def invalidate_rollen_cache(zaak: Zaak):
     _cache = caches["request"]
     if _cache:
-        _cache.delete(f"report:{zaak.url}")
-
-    if rollen:
-        cache_keys = [f"rol:{rol}" for rol in rollen]
-        cache.delete_many(cache_keys)
```
Comment (annashamray) [inline backend/src/zac/core/cache.py:81]: this function is also called when the notification about destroying the rol is received. I think in this case caches for particular rollen `"rol:{rol}"` should also be deleted

**Review by annashamray — CHANGES_REQUESTED** [review]

File: backend/src/zac/core/cache.py (hunk 1 above)
Comment (damm89) [reply backend/src/zac/core/cache.py:81]: If the rol gets destroyed, its cache will be invalidated by the function that destroys it. Should it be moved?

Comment (annashamray) [reply backend/src/zac/core/cache.py:81]: the cache would be cleared if the rol is deleted in the ZAC. But what if it was deleted somewhere else? (like in OpenZaak admin). In this case we will know about it only via notifications

Comment (damm89) [reply backend/src/zac/core/cache.py:81]: You're right, thanks.

**Review by sergei-maertens — APPROVED** [review]
Looks fine, just fix the build before merging :-)

**Review by sergei-maertens — APPROVED** [review]

**Pull request merged**